Source/WebCore/dom/Node.h

```diff
@@ -200,6 +201,8 @@ class Node : public EventTarget {
     WEBCORE_EXPORT const AtomString& lookupPrefix(const AtomString& namespaceURI) const;
     WEBCORE_EXPORT const AtomString& lookupNamespaceURI(const AtomString& prefix) const;
 
+    WEBCORE_EXPORT std::optional<std::variant<String, RefPtr<TrustedScript>>> textContentForBinding();
```

Comment:
Can this return `String` or `std::optional<String>` instead of `std::optional<std::variant>`? Depends on what the bindings generator generates for this.

Reply:
Based on initial testing locally it seems it can just be a String return didn't think that would work given the union type. I guess it makes sense that you can do it, String is ultimately just a more specific subset of the union type. Will inline it in the header too if it works.

Reply:
Turns out this had to be std::optional rather than just String, so I won't inline in header? But at least it's not a variant type anymore.

Reply:
I would still suggest putting it in the header and inlined if you can do that without adding includes to the header.

Reply:
Inlined with or without the inline keyword? Just so I know. I'll let the current push ride the pipelines to check there's nothing else to fix before I push this up.